node/service/src/lib.rs

```diff
@@ -579,6 +592,17 @@ pub fn new_full<RuntimeApi, Executor>(
 	#[cfg(feature = "real-overseer")]
 	config.network.extra_sets.extend(polkadot_network_bridge::peer_sets_info());
 
+	config.network.extra_sets.push(sc_network::config::NonDefaultSetConfig {
+		notifications_protocol: beefy_gadget::BEEFY_PROTOCOL_NAME.into(),
+		max_notification_size: 1024 * 1024,
+		set_config: sc_network::config::SetConfig {
+			in_peers: 25,
+			out_peers: 25,
+			reserved_nodes: Vec::new(),
+			non_reserved_mode: sc_network::config::NonReservedPeerMode::Accept,
+		},
+	});
+
```

Comment:
This could be replaced with `config.network.extra_sets.push(sc_finality_grandpa::grandpa_peers_set_config());`